app/assets/javascripts/proxy_status/dhcp.js

```diff
@@ -0,0 +1,63 @@
+function activateDhcpTables() {
```

Comment:
I am afraid I cannot do namespacing, because [on-complete](https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/application.js#L126) callback just can't handle it:
```javascript
window['foreman.tools.dhcp.activateTables'] != window['foreman']['tools']['dhcp']['activateTables']
```